### DISCUSSION TITLE
Which commit changed svelte compile result

### DISCUSSION THREAD

**Discussion by kingzez** [BuilderIO/jsx-lite#1166] [Q&A] [answered]
I found that the compilation result of svelte has changed, Which commit changed svelte compile result，and I would like to ask why this change. 
```ts
<script context="module" lang="ts">
```

changed to 

```ts
<script lang="ts">
```

**Comment by samijaber** [comment, marked as answer]
I don't think we ever did that, but we could add the change. I don't think there's a downside to having `context="module"` as the default behaviour.

**Reply by samijaber** [reply]
Or even better, only add it if we have a non-empty `json.exports`
